scripts/apps/users/directives/UserListDirective.js

```diff
@@ -25,6 +25,15 @@ export function UserListDirective(keyboardManager, usersService, asset) {
                 bindKeys();
             };
 
+            scope.$watch('users', (users) => {
```

Comment:
I would rather put this logic into `UserListController` where is the rest of the filtering (active/online/etc) so it's all in one place